lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -2034,7 +2034,7 @@ def test_certificate_generation_for_students(self):
             'failed': 0,
             'skipped': 2
         }
-        with self.assertNumQueries(157):
+        with self.assertNumQueries(174):
```

Comment:
This is a pretty decent spike is queries do we understand why this went up so much?